src/graph/create-relation-type.ts

```diff
@@ -0,0 +1,44 @@
+import { ATTRIBUTE, NAME_ATTRIBUTE, SCHEMA_TYPE, TYPES_ATTRIBUTE } from '../core/ids/system.js';
+import { generate } from '../id.js';
+import { Relation } from '../relation.js';
+import { Triple } from '../triple.js';
+import type { CreateRelationOp, Op, SetTripleOp } from '../types.js';
+
+type Params = {
+  from: string;
+  to: string;
+  name: string;
+};
+
+export const createRelationType = async ({ from, to, name }: Params) => {
```

Comment:
@baiirun the `from` and `to` are basically unused atm. I don't know if we even need them or and what triples should be set.

If I look at this type it doesn't seem to define `from` and `to` https://www.geobrowser.io/space/25omwWh6HYgeRQKCaSpVpa/U1uCAzXsRSTP4vFwo1JwJG

Only at the actual relation they seem to be defined: https://www.geobrowser.io/space/6tfhqywXtteatMeGUtd5EB/VbYuNDM2S9LKyij73Aj5CA

Reply:
What is `createRelationType` meant to do?